Dockerfile

```diff
@@ -6,18 +6,20 @@ RUN python -m venv $VIRTUAL_ENV
 ENV PATH "$VIRTUAL_ENV/bin:$PATH"
 WORKDIR /app
 RUN pip install poetry
-EXPOSE 5000
 COPY poetry.lock pyproject.toml /app/
 RUN poetry install --no-root
 COPY . /app
 
 FROM base as production
-ENTRYPOINT poetry run gunicorn -w 4 -b 0.0.0.0:5000 run:app
+ENV PORT 5000
+ENTRYPOINT 'poetry run gunicorn -w 4 -b 0.0.0.0:$PORT "run:app"'
```

Comment:
This command works for Heroku but not for docker locally - when I run it I get:
`/bin/sh: 1: poetry run gunicorn -w 4 -b 0.0.0.0:$PORT "run:app": not found`

Probably the easiest way to align those is to use a shell script as your entrypoint that calls this command

Reply:
Should be fixed now, let me know if you are ok with the changes